src/app/ReadHandler.h

```diff
@@ -217,6 +217,8 @@ class ReadHandler : public Messaging::ExchangeDelegate
     uint16_t mMinIntervalFloorSeconds   = 0;
     uint16_t mMaxIntervalCeilingSeconds = 0;
     SessionHolder mSessionHandle;
+    // only for subscription, mHoldReport is set to true on report delivery within min interval, and then allow to send report after
+    // min interval. if there is urgent event delivery, mHoldReport is also set as false.
```

Comment:
I don't understand this comment.  Is something outside this class setting it?  Is this class setting it?  When it's set to true, what does that mean?  If it's set to false, what does that mean?  Does this change dynamically, or is it set once?

Reply:
updated.
mHoldSync is set to true when trying to refresh subscription timer, which mean empty sync report cannot be sent during max interval. mHoldSync is set to false after max interval is passed, it means empty sync report is
 allowed to be generated.